src/clients/api/queries/getXvsBridgeFeeEstimation/index.spec.ts

```diff
@@ -0,0 +1,48 @@
+import BigNumber from 'bignumber.js';
+import { BigNumber as BN } from 'ethers';
+
+import fakeAddress from '__mocks__/models/address';
+
+import { layerZeroChainIds } from 'constants/layerZero';
+import { XVSProxyOFTSrc } from 'packages/contracts';
+import { ChainId } from 'types';
+
+import getXvsBridgeFeeEstimation from '.';
+
+const fakeAmountMantissa = new BigNumber('1234000');
+
+const fakeSendEstimateFeeParams = [
+  layerZeroChainIds[ChainId.BSC_TESTNET],
+  '0x0000000000000000000000003d759121234cd36F8124C21aFe1c6852d2bEd848',
+  fakeAmountMantissa.toFixed(),
+  false,
+  '0x000100000000000000000000000000000000000000000000000000000000000493E0',
+];
+
+describe('api/queries/getXvsBridgeFeeEstimation', () => {
```

Comment:
I know most tests currently include the path to the file in their name, but I'd actually advise moving away from this as it is unnecessary. It was originally added to make sure the names of the tests were unique, but that's not necessary as vitest groups them by file anyway.
```suggestion
describe('getXvsBridgeFeeEstimation', () => {
```